spec/system/candidate_interface/candidate_entering_degrees_spec.rb

```diff
@@ -379,4 +388,35 @@ def then_i_can_check_my_answers
   def then_i_am_told_i_need_to_add_a_degree_to_complete_the_section
     expect(page).to have_content 'You cannot mark this section complete without adding a degree.'
   end
+
+  def and_i_confirm_i_have_completed_my_degree
+    choose 'Yes'
+    and_i_click_on_save_and_continue
+  end
+
+  def when_i_click_to_change_my_completion_status
+    click_change_link('completion status')
+  end
+
+  def then_i_can_change_my_completion_status
+    expect(page).to have_content 'Have you completed your degree?'
+    choose 'No'
+    and_i_click_on_save_and_continue
+    completion_status_row = page.all('.govuk-summary-list__row').find { |r| r.has_link? 'Change completion status' }
+    expect(completion_status_row).to have_content 'No'
+  end
+
+private
+
+  def click_change_link(row_description)
+    link_text = "Change #{row_description}"
+    page.all('.govuk-summary-list__actions')
+      .find { |row| row.has_link?(link_text) }
+      .click_link(link_text)
+  end
+
+  def expect_validation_error(message)
```

Comment:
Is it worth extracting this into a helper? It's really nice and would be good to use it more widely across the system specs.

Reply:
Definitely! I have a followup PR in the works with some refactors, including this and another spec helper.